org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -3008,6 +3108,24 @@ public void sort(Comparator<Object> cmpr, boolean ascending) {
 		renderItems();
 	}
 
+	/**
+	 * Get table name from AD_Ref_Table of Column Name
+	 * @param refValID
+	 * @param columnName
+	 * @return MTable[] tables
+	 */
+	private MTable getTable(int refValID, String columnName) {
+		if(refValID > 0) {
+			return MTable.get(Env.getCtx(), MRefTable.get(Env.getCtx(), refValID).getAD_Table_ID());
+		}
+		else if (columnName.endsWith("_ID")) {
+			return MTable.get(Env.getCtx(), columnName.substring(0, columnName.length() - 3));
+		}
+		else {
```

Comment:
with the uuid only table changs, I think you can have columnName ends with _UU here as well.